src/main/java/org/wso2/carbon/connector/amazons3/operations/ObjectOperations.java

```diff
@@ -113,7 +114,8 @@ public final void connect(final MessageContext messageContext) throws ConnectExc
                 storageClass, websiteRedirectLocation, ssekmsKeyId, ssekmsEncryptionContext, tagging, objectLockMode,
                 objectLockLegalHoldStatus, copySourceIfMatch, copySourceIfNoneMatch, metadataDirective,
                 taggingDirective, destinationKey, expires, copySourceIfModifiedSince, copySourceIfUnmodifiedSince,
-                objectLockRetainUntilDate, destinationFilePath, fileContent, signatureDurationInMins, isContentAsBase64;
+                objectLockRetainUntilDate, destinationFilePath, fileContent, isContentBase64Encoded,
```

Comment:
Shall we change the variable name here? Since we already have a variable `isContentAsBase64`, this may be ambiguous.

Reply:
Renamed the variable name.